lesson_02/ht_template/job1/tests/test_dal/test_sales_api.py

```diff
@@ -3,14 +3,27 @@
 # TODO: write tests
 """
 from unittest import TestCase, mock
-
 # NB: avoid relative imports when you will write your code:
 from lesson_02.ht_template.job1.dal.sales_api import get_sales
 
 
 class GetSalesTestCase(TestCase):
     """
     Test sales_api.get_sales function.
-    # TODO: implement
     """
-    pass
\ No newline at end of file
+    @mock.patch("lesson_02.ht_template.job1.dal.sales_api.requests")
+    def test_get_sales_success(self, requests_mock):
+        requests_response_mock = mock.MagicMock()
+        requests_response_mock.status_code = 200
+        requests_mock.get.return_value = requests_response_mock
```

Comment:
This can be replaced by 1 line:
```
requests_mock.get.return_value = 200
```

Reply:
also, you may mock the return data in the response, not just a return code. 
Also, it'd be good to emulate API pagination